### ISSUE TITLE
Sometimes normalization brakes twitter intent links 

### ISSUE THREAD

**Issue by 7nik** [r888888888/danbooru#4065]
The ending of some normalized twitter intent links is trimmed.
Examples: 
[raivis draka](https://danbooru.donmai.us/artists/156646)
```
"url": "https://twitter.com/intent/user?user_id=881080941443239936",
"normalized_url": "http://twitter.com/intent/",
```
[miura-n315](https://danbooru.donmai.us/artists/126661)
```
"url": "https://twitter.com/intent/user?user_id=2986344959",
"normalized_url": "http://twitter.com/intent/",
```
While others are ok:
[naribee (nton1263)](https://danbooru.donmai.us/artists/166301)
```
"url": "https://twitter.com/intent/user?user_id=3178955095",
"normalized_url": "http://twitter.com/intent/user?user_id=3178955095/",
```
[beni kurage](https://danbooru.donmai.us/artists/154131)
```
"url": "https://twitter.com/intent/user?user_id=4655268356",
"normalized_url": "http://twitter.com/intent/user?user_id=4655268356/",
"url": "https://twitter.com/intent/user?user_id=876656121552838656",
"normalized_url": "http://twitter.com/intent/user?user_id=876656121552838656/",
```

**Comment by BrokenEagle** [comment]
This was only an issue between 2018-08-31 09:59Z and 2018-09-17 13:22Z, when it was broken by a1044db and then later fixed with 325120e. So there's nothing to do code-wise at this point.

One way to fix this would be to refresh the URLs. This can be done by simply removing the affected URLs, saving the artist entry, then re-adding the URLs, and again saving the artist entry.

Another option would be to fix them server-side with a fix script.

**Comment by evazion** [comment]
Added a fix script to fix the remaining incorrect URLs. This fixes about ~1300 URLs. This search can be used to find incorrect URLs:

https://danbooru.donmai.us/artist_urls?&search[normalized_url_matches]=http://twitter.com/intent/
